cves/CVE-2009-2901.yml

```diff
@@ -116,10 +127,12 @@ discovered:
     If there is no evidence as to how this vulnerability was found, then you
     may leave the entries blank except for "answer", BUT please write down
     where you looked in "answer".
-  answer:
-  date:
-  automated:
-  contest:
+  answer: |
+    This find is credited to the Apache Tomcat security team, as seen here -
```

Comment:
I assume this means there is no single person called out specifically?

Reply:
No credit was given to an individual (that I could find anyways). The best answer I could get for a find was "the Apache Tomcat security team," which is what I opted to put here.